modules/manage/partials/tiered-storage.adoc

```diff
@@ -1250,6 +1250,50 @@ endif::[]
 
 == Disaster recovery
 
+With tiered storage enabled, you can restore your cluster, including its metadata, in the event of failure. This is a faster, less expensive alternative to active-active replication (for example, using xref:upgrade:migrate/data-migration.adoc[MirrorMaker 2]).
+
+You can take the following steps to restore a failed cluster:
+
+. Start a new cluster
+. Recover metadata from a failed source cluster to the new cluster
+. Recover topic data from the source cluster to the new cluster
+. Perform a rolling restart, if necessary
+. Direct application workload to the new cluster
+
+Use this recovery method to restore your application to the latest functional state as quickly as possible. Note that this method does not guarantee consistency at the application level. Restored transactions may not uphold atomicity. The consumer offset topic may not reflect point-in-time snapshots of topic data already consumed by applications. 
+
+=== Start a recovery target cluster
+
+Follow the steps to xref:deploy:deployment-option/cloud/index.adoc[provision a new cloud cluster]. Cloud account credentials (such as secrets or access keys) are not included in backed up metadata, and so you must manually configure these credentials for the new cluster.
```

Comment:
@kbatuigas this should not be assuming Cloud.  We should say something like "if using Redpanda Cloud, follow this doc.  If using Redpanda Enterprise, follow this doc (self-hosted deploy docs)"

Reply:
@andrwng should we link to HOW to configure cloud credentials as cluster properties?  And also that if using IAM roles this isn't necessary?

Reply:
Yeah, perhaps https://docs.redpanda.com/current/manage/tiered-storage/#set-up-tiered-storage ?

Reply:
>And also that if using IAM roles this isn't necessary?

@mattschumpert actually I'm confused by this, can you clarify? What changes if IAM roles are configured? The recovery right now doesn't restore cloud storage credentials and configs, with the assumption that the cluster is already able to access the cloud.

This makes me think we should be restoring the [cloud_storage_credentials_source](https://docs.redpanda.com/current/reference/cluster-properties/#cloud-storage) property (we aren't now, but I can update that). Is that right?

Reply:
I chatted a bit with @abhijat and I think the original stance is sound: cloud credentials (whether that be IAM or keys) should be configured before restoring. If this isn't configured in the first place, a restore wouldn't be possible because we'd be unable to access the bucket.